pipeline/post-results-new.groovy

```diff
@@ -0,0 +1,278 @@
+// Pipeline script for uploading IBM test run results to report portal.
```

Comment:
is this going to be separate job ? Then IMO we should parameterize it with job id and repo url

Reply:
No it is the same post-results job, i created a different groovy just to test out things.